pkg/browsers/browsers.go

```diff
@@ -25,7 +26,7 @@ var BravePathWindows = []string{`\Program Files\BraveSoftware\Brave-Browser\Appl
 var EdgePathMac = []string{"/Applications/Microsoft Edge.app/Contents/MacOS/Microsoft Edge"}
 var EdgePathLinux = []string{`/usr/bin/edge`}
 var EdgePathWindows = []string{`\Program Files (x86)\Microsoft\Edge\Application\msedge.exe`}
-var FirefoxPathMac = []string{"/Applications/Firefox.app/Contents/MacOS/firefox"}
+var FirefoxPathMac = []string{"/Applications/Firefox.app/Contents/MacOS/firefoxs"}
```

Comment:
Do we need to change this back? Im guessing you added the s for testing when its not installed